pkg/handler/enqueue_mapped.go

```diff
@@ -36,33 +34,41 @@ var _ EventHandler = &EnqueueRequestsFromMapFunc{}
 //
 // For UpdateEvents which contain both a new and old object, the transformation function is run on both
 // objects and both sets of Requests are enqueue.
-type EnqueueRequestsFromMapFunc struct {
+func EnqueueRequestsFromMapFunc(mapFN func(MapObject) []reconcile.Request) EventHandler {
```

Comment:
```suggestion
func MapFunc(mapFN func(MapObject) []reconcile.Request) EventHandler {
```

Do we want to go for something a little simpler? wdyt?

Reply:
All existing mappers are called `EnqueueRequestSomething` I thought it would be nice to keep that pattern

Reply:
It also makes this change easier for existing users